js/form.js

```diff
@@ -0,0 +1,132 @@
+                        {
+                            id: eventID,
+                            name: eventName,
+                            description: eventDescription,
+                            date: eventDate
+                        });
+                    window.location.href = "list.html";
+                });
+            }
+            else {   //To edit an event
+                document.getElementById("heading1").innerHTML = "Edit Event";
+                document.getElementById("submit-form-btn").value = "Save";
+                document.getElementById("event-name").value = name;
+                document.getElementById("event-description").value = description;
+                document.getElementById("event-date").value = date;
+                submitBtn.addEventListener("click", (event) => {
+                    event.preventDefault();
+                    let newName = document.getElementById("event-name").value;
+                    let newDescription = document.getElementById("event-description").value;
+                    let newDate = document.getElementById("event-date").value;
+                    newName = newName.charAt(0).toUpperCase() + newName.substr(1).toLowerCase();//convert to title case
+                    set(ref(db, 'events/' + id),
+                        {
+                            id: id,
+                            name: newName,
+                            description: newDescription,
+                            date: newDate
+                        });
+                    set(ref(db, 'eventToEdit/'),
+                        {
+                            id: "none",
+                            name: "none",
+                            description: "none",
+                            date: "none"
+                        });
+                    window.location.href = "list.html";
+                });
+            }
+        } else {
+            console.log("No data available");
+        }
+    }).catch((error) => {
+        console.error(error);
+    });
+}
+
+//This function is used to validate form inputs
+function validateForm() {
+    console.log("i am in validation");
+    let name = document.getElementById("event-name").value;
+    let date = document.getElementById("event-date").value;
+    if (name && date) {
+        if (name == null || name == "" || name == " ") { //|| name.trim()==""
+            //alert("Name can't be blank");
+            submitBtn.disabled = true;
+            return false;
+        }
+        else if (name.length < 3) {
+            //alert("Name must be at least 3 characters");
+            submitBtn.disabled = true;
+            return false;
+        }
+        else {
+            //alert('Valid name given.');
+            submitBtn.disabled = false;
+            return true;
+        }
+    }
+    else {
+        //alert('Empty name or date given.');
+        submitBtn.disabled = true;
+        return false;
+    }
```

Comment:
please reduce this logic to
 submitBtn.disabled = name == null || name == "" || name == " " || name.length < 3 || !data;

Reply:
done